deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+---
+{{if .Values._000_commonConfig.datasources.barcodereader.enabled}}
```

Comment:
these changes are probably not wanted. rebase?

Reply:
I have no idea, why git is even tracking file permissions, especially on windows.
But tbh dropping the executable bit on those files seems fine to me